### ISSUE TITLE
last.fm

### ISSUE THREAD

**Issue by Botanistt** [DiNitride/Discord-Music-Status#43]
Hey. Can you add last.fm support? I would like to use it instead of all the other available options.

**Comment by DiNitride** [comment]
Define last.fm support, last.fm isn't a media player afaik

**Comment by Botanistt** [comment]
Yeah, it is not but it tracks music you are listening to from various platforms like youtube, spotify, soundcloud, and a lot more.
example of my account https://www.last.fm/user/wueue
You can get the data via json.

**Comment by DiNitride** [comment]
Right now I am too busy with college but it's a neat idea so I'll consider it for a future update

**Comment by DiNitride** [comment]
why did i close it im dumb

**Comment by Botanistt** [comment]
¯\_(ツ)_/¯

**Comment by hrueschwein** [comment]
Hello, @Botanistt! I'm making utility to show in status what are you scrobbling. If you're interested, look: [dikey0ficial/dlfm](https://github.com/dikey0ficial/dlfm)
